connector/src/main/java/org/geysermc/connector/utils/PluginMessageUtils.java

```diff
@@ -46,6 +51,8 @@
                         .array();
 
         FLOODGATE_REGISTER_DATA = (SKIN_CHANNEL + "\0floodgate:form").getBytes(Charsets.UTF_8);
+
+        EMOTE_REGISTER_DATA = (EMOTE_CHANNEL + "\0").getBytes(StandardCharsets.UTF_8);
```

Comment:
Can this not use `Charsets.UTF_8` or vice versa to save the extra import for the same thing?
Probably best to use `StandardCharsets` since its part of `nio`